doc/modules/linear_model.rst

```diff
@@ -1010,10 +1013,15 @@ a matrix of coefficients :math:`W` where each row vector :math:`W_k` corresponds
 
 The objective for the optimization becomes
 
-.. math:: \min_W -C \sum_{i=1}^n \sum_{k=0}^{K-1} [y_i = k] \log(\hat{p}_k(X_i)) + r(W).
+.. math::
+  \min_W -\frac{1}{S}\sum_{i=1}^n \sum_{k=0}^{K-1} s_{ik} [y_i = k] \log(\hat{p}_k(X_i))
+  + \frac{r(W)}{S C}\,.
 
 Where :math:`[P]` represents the Iverson bracket which evaluates to :math:`0`
-if :math:`P` is false, otherwise it evaluates to :math:`1`. We currently provide four choices
+if :math:`P` is false, otherwise it evaluates to :math:`1`.
+Again, :math:`s_{ik}` are the weights assigned by the user (multiplication of sample
+weights and class weights) with their sum :math:`S = \sum_{i=1}^n \sum_{k=0}^{K-1} s_{ik}`.
+We currently provide four choices
```

Comment:
I feel like the goal of the introduced line breaks was to have line breaks in the rendered doc. For that I think we need a blank line in between.
```suggestion

Again, :math:`s_{ik}` are the weights assigned by the user (multiplication of sample
weights and class weights) with their sum :math:`S = \sum_{i=1}^n \sum_{k=0}^{K-1} s_{ik}`.

We currently provide four choices
```